docs/ABI/Mangling.rst

```diff
@@ -531,8 +531,8 @@ Types
   FUNCTION-KIND ::= 'B'                      // objc block function type
   FUNCTION-KIND ::= 'zB' C-TYPE              // objc block type with non-canonical C type
   FUNCTION-KIND ::= 'L'                      // objc block function type with canonical C type (escaping) (DWARF only; otherwise use 'B' or 'zB' C-TYPE)
-  FUNCTION-KIND ::= 'C'                      // C function pointer type
-  FUNCTION-KIND ::= 'zC' C-TYPE              // C function pointer type with with non-canonical C type
+  FUNCTION-KIND ::= 'C'                      // C function pointer / method type
+  FUNCTION-KIND ::= 'zC' C-TYPE              // C function pointer / method type with with non-canonical C type
```

Comment:
Please use "C++ method" (otherwise, it's not clear if this is referring to ObjC methods or C++ methods).

Reply:
Thanks, done.